tutorials/evoked/30_eeg_erp.py

```diff
@@ -373,7 +374,204 @@
 #
 # The tutorials :ref:`tut-epochs-class` and :ref:`tut-evoked-class` have many
 # more details about working with the `~mne.Epochs` and `~mne.Evoked` classes.
+
+# %%
+# Amplitude and Latency Measures
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+# It is common in ERP research to extract measures of amplitude or latency to
+# compare across different conditions. There are many measures that can be
+# extracted from ERPs, and many of these are detailed (including the respective
+# strengths and weaknesses) in Ch. 9 of Luck :footcite:`Luck2014` (also see
+# the `Measurement Tool <https://bit.ly/37uydRw>`_ in the ERPLAB Toolbox
+# :footcite:`Lopez-CalderonLuck2014`).
+#
+# This part of the tutorial will demonstrate how to extract three common
+# measures:
+#
+# * Peak Latency
+# * Peak Amplitude
+# * Mean Amplitude
+#
+# Peak Latency and Amplitude
+# --------------------------
+#
+# Probably most common measures of amplitude and latency are peak measures.
+# Peak measures are basically the maximum amplitude of the signal in a
+# specified time window, and the time point (or latency) at which the peak
+# amplitude occurred.
+#
+# Peak measures can be obtained using the `~mne.Evoked.get_peak` method. There
+# are two important things to point out about `~mne.Evoked.get_peak` method.
+# First, it returns the peak latency and amplitude from **all channels** in
+# the `~mne.Evoked` object.
+# Second, the `~mne.Evoked.get_peak` method can find different 'types' of
+# peaks using the ``mode`` argument. There are three options:
+#
+# * ``mode='pos'``: finds the peak with a positive voltage (ignores
+#   negative voltages)
+# * ``mode='neg'``: finds the peak with a negative voltage (ignores
+#   positive voltages)
+# * ``mode='abs'``: finds the peak with the largest absolute voltage
+#   regardless of sign (positive or negative)
```

Comment:
```suggestion
# * ``mode='pos'`` finds the peak with a positive voltage
# * ``mode='neg'`` finds the peak with a negative voltage
# * ``mode='abs'`` finds the peak with the largest absolute voltage
#   regardless of sign (positive or negative)
```

Reply:
to me "finds the peak with a positive voltage" is not clear (there may be many peaks with positive voltage, which one is *the* one?) Suggested alternative: "finds the highest positive-voltage peak"

Reply:
This makes me wonder if `get_peak()` is actually finding a peak at all – after all, it is just returning the max/min of a given array. Is this something we should discuss in another issue?

Reply:
I haven't scrutinized the implementation, but based on the example below I agree it does seem to be doing a fairly naive min/max operation.  I think that at least deserves a `Notes` section in the docstring of the method, but let's do that in a separate issue/PR.

Reply:
Cf #9681.